VlsirTools/vlsirtools/netlist/spice.py

```diff
@@ -143,6 +143,9 @@ def write_instance_name(
     def write_instance(self, pinst: vckt.Instance) -> None:
         """Create and return a netlist-string for Instance `pinst`"""
 
+        if not pinst.HasField("module"):
```

Comment:
What's this do? 
Is that like a semi-recently deprecated field name?

Reply:
(From experience) if your incoming vckt.Package is garbage and has Instances that don't refer to Modules (i.e. the Module field is empty), `write_instance` fails during the call to `resolve_reference`, and it isn't really clear why. This just provides a helpful error for users feeding in the garbage to know what to do about it.

Reply:
I see, it's just like the null-check before passing to something that expects an (actual) `Ref`. Makes sense!